src/hooks/useNotifications.tsx

```diff
@@ -309,6 +309,15 @@ const useNotificationsContext = () => {
   // Menu state
   const [isMenuOpen, setIsMenuOpen] = useState(false);
 
+  // Unread
+  const hasUnreadNotifications = useMemo(
+    () =>
+      Object.values(notifications).filter(
+        (notification) => notification.status < NotificationStatus.Seen
+      ).length > 0,
```

Comment:
nit: can we do `Object.values(notifications).some((notification) => notification.status < NotificationStatus.Seen)`?